src/main/java/com/smartling/marketo/sdk/MarketoFormClient.java

```diff
@@ -29,6 +30,14 @@
 
     void updateFormField(int formId, FormField formField) throws MarketoApiException;
 
+    void discardFormDraft(int formId) throws MarketoApiException;
+
+    void deleteFormField(int formId, String fieldId) throws MarketoApiException;
+
+    FormField addFormRichTextField(int formId, String text) throws MarketoApiException;
+
+    void reArrangeFormFields(int formId, UpdateFieldPositionsList positions) throws MarketoApiException;
```

Comment:
Why reArrange instead of rearrange?

Reply:
I keep original notation from marketo api doc